### PR TITLE
Quickstart: DataOwner: Join Network

### LINKED ISSUE #6688: GSoD: Quickstart notebooks for data owners
## Where?
We would like to extend the quickstart section of the documentation with the main remaining tutorials for the data owners:

1. Create Network (@ShubhamPalriwala )  - #6800 
    - Learning objective: How data can be shared among multiple people, the concept of a network
2. Join a network (@ShubhamPalriwala ) - #6693 
    - Learning Objective: How to discover networks and contribute to them
3. Create Accounts + Manage Privacy Budget (@abhiwalia15 ) - #6759 #6789 
    - Learning objective: How to manage the access to your domain and uploaded dataset and why is it important

The first iteration of all notebooks should be ready for review by Friday (2nd).

## Who?
New users to OpenMined looking to learn how to use PySyft and PyGrid as a data owner.

## What?

- [x] Define create a network, join a network flow 
- [x] Define functionalities to be showcased in the management of the privacy budget
- [x] Finish notebook tutorials in notebooks/quickstart
   - [x] Create a network
   - [x] Join a network
   - [x] Create account + manage privacy budget 
- [ ] Add corresponding pages to docs/ section
   - [x] Create a network
   - [x] Join a network
   - [ ] Create account + manage privacy budget
- [ ] Link tutorials in the README.md

Quality control (Linting and testing):
- [ ] Make sure the notebooks run
- [ ] Keep an eye out for formatting of code in cells
- [ ] No excess spacing on headings
- [ ] Use normal font size for main text and custom headings
- [ ] Proofread spelling and punctuation
- [ ] Check for consistency

### PR DESCRIPTION
**Pull request by ShubhamPalriwala**
## Description
Partially closes #6688 

- [x] Initialise with basic commands that are already known with 1 liner descriptions 
- [x] Command for Logging in into a network with a custom URL 
- [x] Functionality Testing
- [x] Link to previous and next notebook
- [x] Add base64 encoded images
- [x] All UI changes
- [x] Add Title
- [ ] Add Banner Image

## Affected Dependencies
None

## How has this been tested?
- [x] Thorough Testing
- [ ] Corner cases

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: notebooks/quickstart/data-owner/03-join-network.ipynb [hunk 1]
```diff
@@ -0,0 +1,359 @@
+{
```
Comment (shubham3121) [inline notebooks/quickstart/data-owner/03-join-network.ipynb:1]: <p>Can we also define in text before this cell that the default admin credentials to the domain are:</p><ul><li>Email: <code>info@opemined.org</code></li><li>Password: <code>changethis</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6693/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (shubham3121) [inline notebooks/quickstart/data-owner/03-join-network.ipynb:1]: <p>sounds incomplete: .. change it with any other available Network.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6693/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (ShubhamPalriwala) [reply notebooks/quickstart/data-owner/03-join-network.ipynb:1]: It is there already just above the cell, "The below cell has default credentials, please change accordingly."
Let me know if you meant rephrasing/changing it?

[the remark this answers is not in the document]
Comment (ShubhamPalriwala) [reply notebooks/quickstart/data-owner/03-join-network.ipynb:1]: Renamed to "OpenMined TestNet" as discussed in the Slack thread

[the remark this answers is not in the document]
Comment (ShubhamPalriwala) [reply notebooks/quickstart/data-owner/03-join-network.ipynb:1]: We have mentioned this in the docs page that goes up on the web for this notebook. If you still think we need this, then maybe we can add it to all the notebooks to maintain consistency?

**Review by IrinaMBejan — APPROVED** [review]

**Comment by IrinaMBejan** [comment]
Great work @ShubhamPalriwala!

**Pull request merged**